### PR TITLE
Develop more array vehicle tests jvbb

### PR DESCRIPTION
**Pull request by JVBB83**

### CODE REVIEW

File: BusinessCore.Tests/BusinessCore.Tests.csproj [hunk 1]
```diff
@@ -51,6 +51,7 @@
   <ItemGroup>
     <Compile Include="alloretTests.cs" />
     <Compile Include="amunozeverisTests.cs" />
+    <Compile Include="JVBB-FunctionalProgramingTests %28OF DOOM%29.cs" />
```
Comment (hossmi) [inline BusinessCore.Tests/BusinessCore.Tests.csproj]: Please, rename without special characters

File: BusinessCore.Tests/JVBB-FunctionalProgramingTests (OF DOOM).cs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+        public JVBBFunctionalProgramingTests()
+        {
+            this.vehicleStorage = new JVBBArrayVehicleStorage();
+        }
+
+        [TestMethod]
+        public void all_vehicles_with_enrollment_serial_CSM_have_666_horsepower()
+        {
+            bool isTrue = this.vehicleStorage
```
Comment (hossmi) [inline BusinessCore.Tests/JVBB-FunctionalProgramingTests (OF DOOM).cs]: test must compile

File: BusinessCore.Tests/BusinessCore.Tests.csproj (hunk 1 above)
Comment (JVBB83) [reply BusinessCore.Tests/BusinessCore.Tests.csproj]: Done

File: BusinessCore.Tests/JVBB-FunctionalProgramingTests (OF DOOM).cs (hunk 2 above)
Comment (JVBB83) [reply BusinessCore.Tests/JVBB-FunctionalProgramingTests (OF DOOM).cs]: It compiles now

**Pull request merged**